seed/seed.json

```diff
@@ -1,5 +1,48 @@
 {
     "studies": [
+        {
+            "experiments": [
+                {
+                    "feature_association": {
+                        "enable_feature": [
+                            "AIChat"
+                        ]
+                    },
+                    "name": "DefaultLlama",
+                    "params": {
+                        "AIChat": {
+                            "default_model": "chat-basic"
+                        }
+                    },
+                    "probability_weight": 25
+                },
+                {
+                    "feature_association": {
+                        "enable_feature": [
+                            "AIChat"
+                        ]
+                    },
+                    "name": "DefaultMixtral",
+                    "probability_weight": 75
+                }
+            ],
+            "filter": {
+                "channel": [
+                    "RELEASE",
+                    "BETA",
+                    "NIGHTLY"
+                ],
+                "min_version": "122.0.6261.57",
```

Comment:
looks like in this file the versioning is [chrome_major].[brave_version], e.g `122.1.63.0`?

Reply:
I updated to use `122.1.63.161` as the min version of the default model change. 1.63.161 seems to be the first 1.63 release. This also adds in a little buffer because https://github.com/brave/brave-browser/issues/34721 was uplifted into 1.62.x.

I specified `122.1.63.160` as the new max version for the previous setting. This may not correlate to an actual release, it's just one patch version behind `122.1.63.161`.